README.md

```diff
@@ -2,9 +2,13 @@
 
 This repository supports the GitHub Pages site for jclouds. See and read more at [http://www.jclouds.org](http://www.jclouds.org).
 
+To test the site locally:
+
+    jekyll _1.5.1_ serve --safe
```

Comment:
`jekyll <version> serve --safe`?

Reply:
Not italics, but the Ruby syntax to use a concrete version of the gem! :smile:

Reply:
Aaaaaah ;-) Thanks for explaining!